app/login/page.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
```

Comment:
M: Unless this is named 'test' for a particular reason, please change this to be a little more descriptive/relative to what you're testing.